package.json

```diff
@@ -127,8 +128,9 @@
         "karma-firefox-launcher": "^1.3.0",
         "karma-mocha": "^2.0.1",
         "karma-spec-reporter": "0.0.32",
-        "lerna": "^4.0.0",
+        "lerna": "^6.1.0",
```

Comment:
Moving from lerna 4 to 6, I think [life cycle scripts](https://github.com/lerna/lerna/tree/main/commands) remain same, could you also double check?  
https://github.com/lerna/lerna/tree/main/commands/publish
https://github.com/lerna/lerna/tree/main/commands/version
https://github.com/lerna/lerna/tree/main/commands/run

a part from these few discussion items, this PR looks good, Thanks Alex

Reply:
Looking from 4 to 6, there hasnt been any changes within the script. So it has remained the same